temporaldata/typing.py

```
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+from typing import Protocol, TypeAlias
+
+import numpy as np
+
+
+@runtime_checkable
```

Ruff (0.15.13)

[error] Undefined name `runtime_checkable`

(F821)